CHANGELOG.md

```diff
@@ -20,6 +20,14 @@ _None_
 
 _None_
 
+## 2.2.1
```

Comment:
Please move your entry under `## Master` — that's what it's there for — rather than assuming that the next version will be a 2.2.1 (we might not release a new version immediately so maybe there will be other PRs by other people that will make us need to bump the minor version when we decide to release) :wink: